frontend/src/tests/utils.js

```diff
@@ -175,7 +176,7 @@ function _ensurePath(path) {
  * @returns
  */
 export async function apiGET(url, omitPrefix = false) {
-    url = omitPrefix ? _ensurePath(url) : API_URL + _ensurePath(url);
+    url = omitPrefix ? URL + _ensurePath(url) : API_URL + _ensurePath(url);
```

Comment:
Good catch! What do you think of something like

```
const BASE_URL = "http://backend:3000";
....
export async function apiGET(url, omitPrefix = false) {
    url = `${BASE_URL}${omitPrefix ? '' : '/api/v1'}${_ensurePath(url)}`;
    ...
```

Or alternatively:

```
const BASE_URL = "http://backend:3000";
const API_ROUTE = '/api/v1';
....
export async function apiGET(url, omitPrefix = false) {
    let urlParts = [BASE_URL];
    if (!omitPrefix) {
        urlParts.push(API_ROUTE);
    }
    url = urlParts.concat();
    ...
```

Reply:
I prefer the first one, but even better would be to have `addPrefix`, instead of `omitPrefix` (omit = false is double negation), but that's a style choice that is consistent with the real method after all.

Reply:
Look into the standard `URL` object before you reinvent the wheel too much ;-)

Reply:
I forgot of its existence :joy: Something like `url = new URL(omitPrefix ? '' : 'api/v1', 'https://localhost:8000');`? Looks a lot better indeed!